lib/wrapper/slack.go

```diff
@@ -0,0 +1,44 @@
+package wrapper
+
+import (
+	"github.com/robiball/slack"
+	"github.com/adampointer/go-slackbot"
+
+	"fmt"
+)
+
+var(
+	SlackApp *slack.Client
+	SlackBot *slackbot.Bot
+)
+
+// Create connections to Slack
```

Comment:
Please use standard comments for Godoc i.e.

`// Init creates connections to Slack`

Also avoid capitalised args use `appToken` and `botToken`

Reply:
Done